### ISSUE TITLE
Eslint-docs and prettier seem to disagree

### ISSUE THREAD

**Issue by uniqueiniquity** [bradzacher/eslint-plugin-typescript#249]
I'm trying to commit some work I've done on implementing a version of TSLint's `no-for-in-array`. However, upon syncing with `master`, it seems like I can't get past the test hook. Specifically, `yarn docs` keeps deleting the top of `indent.md` and `type-annotation-spacing.md`. They also seem to disagree on whether there should be line breaks in the table for the README. It seems like `eslint-docs` is not behaving correctly, but I wanted to file an issue before overriding the test hook. Which should win, `eslint-docs` or `prettier`?

**Comment by armano2** [comment]
try installing new version of packages :>

**Comment by uniqueiniquity** [comment]
Didn't help 😞 Even tried deleting my `node_modules` and starting from scratch.

**Comment by uniqueiniquity** [comment]
For example (the `indent` one is much longer but similar):
```
× The description for type-annotation-spacing must match the required format:
  --- docs\rules\type-annotation-spacing.md
  +++ generated
  @@ -1,10 +1,5 @@
  -# Require consistent spacing around type annotations (type-annotation-spacing)
  -
  -Spacing around type annotations improves readability of the code. Although the most commonly used style guideline for type annotations in TypeScript prescribes adding a space after the colon, but not before it, it is subjective to the preferences of a project. For example:
  -
  -<!-- prettier-ignore -->
  -```ts
  +# Require consistent spacing around type annotations (type-annotation-spacing)
   // with space after, but not before (default if no option is specified)
   let foo: string = "bar";

   // with no spaces
```

**Comment by j-f1** [comment]
Very strange. What OS are you on? Does running `yarn add --dev eslint-docs@latest prettier@latest` fix the issue?

**Comment by uniqueiniquity** [comment]
I'm on Windows (I've had issues with that in the past...).
Will try the command now.

**Comment by uniqueiniquity** [comment]
The command did not fix the issue.

**Comment by armano2** [comment]
what version of node are you using?

**Comment by uniqueiniquity** [comment]
v9.11.1

**Comment by armano2** [comment]
can you send here your modified `type-annotation-spacing.md` -> i'm unable to reproduce this

**Comment by uniqueiniquity** [comment]
[mangled-docs.txt](https://github.com/bradzacher/eslint-plugin-typescript/files/2692462/mangled-docs.txt)

**Comment by armano2** [comment]
ok, thank you, issue is present when file has line windows line endings for some reason `eslint-docs` is overriding big portion of code, i will try to fix that.

for now please change line ending of file to unix style `\n` instead of `\r\n`

**Comment by uniqueiniquity** [comment]
That was it. Sorry for not checking that sooner, that usually seems to be the issue. I'll close this since it's not really relevant to this repo.
